app/routes/funding-options/land-primary-actions.js

```diff
@@ -59,7 +60,7 @@ module.exports = [
           if (agreement.paymentActions?.length > 0) {
             return h.redirect('land-increased-actions')
           } else {
-            await sendAgreementValidateMessage(agreement, request.yar.id)
+            await sendAgreementValidateMessage(buildMessage(await cache.get('agreement', request.yar.id)), request.yar.id)
```

Comment:
I would break this into two lines. I don't like seeing an "await" in the middle of a line of code, I find it's easy to miss. (This is just an opinion)